features/bootstrap/FeatureContext.php

```diff
@@ -427,4 +427,37 @@ public function iPerformActionsAndChecksForMenuItem($menuItem) {
     print("End item");
   }
 
+
+  /**
+   * Check if X class have X style with X value.
+   *
+   * @param string $selector
+   * @param string $style
+   * @param string $value
+   *
+   * @return null
+   * @throws Exception
+   * @Then I should see that :style with :value is in :selector class
+   */
+  public function cssStyleExists(string $selector, string $style, string $value) {
+    $function = <<<JS
+(function(){
+var accordionItem = document.querySelector('$selector');
+var styles = window.getComputedStyle(accordionItem);
```

Comment:
`$selector` passed in won't always be for the accordion, so let's use a generic name here, like simply `$item`.